SMLHelper/Utility/KeyCodeUtils.cs

```diff
@@ -135,5 +135,124 @@ public static KeyCode StringToKeyCode(string s)
                     }
             }
         }
+
```

Comment:
This all looks pretty good.

One question: Are there any considerations to be made for the small differences between Subnautica and Below Zero?

There probably aren't but I just want to be sure.

Reply:
Given how there are next to no mods doing anything with controller support, I don't think this will be anything to worry about.

I'll set up new branch for the next release later and get this merged.
There's another open PR that needs my attention in the meantime.

Reply:
Yeah no rush. I can confirm btw that it in my testing, it works as intended in BZ.